dxr/plugins/descriptor/__init__.py

```diff
@@ -141,7 +141,10 @@ def generic_describe(self):
 
         # Haven't returned so we can fall back to the first non-empty line of
         # the first doc-comment.
-        # TODO: skip the license
-        match = self.comment_re.search(''.join(self.sixty_lines))
-        if match:
-            return match.group('description')
+        for match in self.comment_re.finditer(''.join(self.sixty_lines)):
+            desc = match.group('description')
+            desc_lower = desc.lower()
+            # Skip any comment that contains the license or a tab-width
+            # emacs/vim setting.
+            if not any((pattern in desc_lower for pattern in {'tab-width', 'license', 'vim:'})):
```

Comment:
There's an extra set of parens here. (A genexpr doesn't need its own set if it's the only arg to a func.) And there's no sense using a set (and the extra hasing and RAM) because we're just iterating over it.